observability/src/main/java/io/grpc/observability/logging/GcpLogSink.java

```diff
@@ -94,6 +117,21 @@ public void write(GrpcLogRecord logProto) {
     }
   }
 
+  @VisibleForTesting
+  MonitoredResource getResource(Map<String, String> resourceTags) {
```

Comment:
The parameter is called `resourceTags` whereas we pass in something called `locationTags`. If `resourceTags` is the better term for it we should consider renaming `locationTags` to `resourceTags` everywhere. Not for this PR other than a TODO comment may be...